tools/fix_auth/fix_auth.rb

```diff
@@ -51,21 +52,13 @@ def generate_password
     end
 
     def fix_database_passwords
-      begin
-        ActiveRecord::Base.connection_config
-      rescue ActiveRecord::ConnectionNotEstablished
-        # not configured, lets try again
-        ActiveRecord::Base.logger = Logger.new("#{options[:root]}/log/fix_auth.log")
-        please_establish_connection = true
-      end
+      log_path = "#{options[:root]}/log/fix_auth.log"
       databases.each do |database|
```

Comment:
Yes, I need this for other "slim worker" items.  I am trying to add it on it's own for now so it is a isolated PR, and extract the functionality in a reusable format where it already exists.

This rational exists at the top of the PR.  Is that not clear?